### PR TITLE
Responsiveness and footer

### PR DESCRIPTION
**Pull request by kotalwar-krishna**
-Footer is added. related css you can find it in footer.css.
-responsiveness with respect to page is added.
- responsiveness of iframe i will add latter.

### CODE REVIEW

**Comment by psychoactiv** [comment]
Its's good to go

**Review by psychoactiv — APPROVED** [review]

**Review by psychoactiv — APPROVED** [review]

**Review by psychoactiv — APPROVED** [review]

File: components/footer.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.footer {
+    background-color: #956cf3;
```
Comment (SuyashTiwariCoder) [inline components/footer.css:2]: add variable

File: components/footer.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    padding: 2rem 1rem;
+    text-align: center;
+    color: white;
```
Comment (SuyashTiwariCoder) [inline components/footer.css:5]: add color variables every where

File: documentation/doc.html [hunk 3]
```diff
@@ -70,6 +70,18 @@ <h3 class="component-title">Components</h3>
...
     <section class="component-information-section">
+      <!-- Use IT -->
+      <div class="component-doc-container">
+        <h1 id="start">How to use Components</h1>
+        <p class="component-information">
+          Copy-paste the stylesheet &lt;link&gt; into your &lt;head&gt; before all other stylesheets to load our CSS.
+        </p>
+        <div>
+          <!-- <iframe src="https://www.thiscodeworks.com/embed/61f908f11e1df400153793cb" style="width: 100%; max-height: 200px;"
+          frameborder="0"></iframe> -->
```
Comment (SuyashTiwariCoder) [inline documentation/doc.html]: remove unnecessary comments

**Review by SuyashTiwariCoder — APPROVED** [review]
You are good to go

File: components/footer.css [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    padding-inline-start: 0px;
+}
+
+.list-non-bullet {
+    list-style: none;
+}
+
+.list-item-inline {
+    display: inline;
+    padding: 0.5rem 0.5rem;
```
Comment (sakleprajwal) [inline components/footer.css:28]: Do not use rem for smaller size requirement

**Review by sakleprajwal — COMMENTED** [review]
Suggested a minor change. Please check.

File: components/footer.css (hunk 4 above)
Comment (kotalwar-krishna) [reply components/footer.css:28]: ok

**Pull request merged**